ArjunaJTS/jts/classes/com/arjuna/ats/internal/jts/orbspecific/ibmorb/recoverycoordinators/JavaIdlRCServiceInit.java

```diff
@@ -247,7 +247,7 @@ private static String initRCKey() {
     }
 
     private static final String POA_NAME_PREFIX = "RcvCo-";
-    private static String OBJ_KEY_PREFIX = ""; // is there an equivalent of JacORBs jacorb.implname
```

Comment:
How about leaving this comment to help with identifying something to use even after we stop supporting use of JacORB?

Reply:
I think that `is there an equivalent of JacORBs jacorb.implname` can lead to confusion. Maybe, we can change this comment to explain what `OBJ_KEY_PREFIX` is used for. WDYT?

Reply:
Thanks